### PR TITLE
docs(spin/pipeline-templates) Adds pipeline template cmd docs.

### PR DESCRIPTION
**Pull request by jtk54**
I apparently didn't actually add the pipeline template docs in the previous PR like I claimed.

### CODE REVIEW

File: guides/spin/pipeline-templates/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Usage:
+   pipeline-template [command]
+
+Aliases:
+  pipeline-template, pipeline-templates, pt
+
+Available Commands:
+  delete      Delete the provided pipeline template
+  get         Get the pipeline template with the provided id
```
Comment (dorbin) [inline guides/spin/pipeline-templates/index.md]: s/id/ID/

File: guides/spin/pipeline-templates/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  ...
+}
+...
+]
+
+```
+
+### Retrieve a single pipeline template with `get`
+
+Get a single pipeline with `get`:
```
Comment (dorbin) [inline guides/spin/pipeline-templates/index.md]: You can delete this sentence, as it's pretty-much already convered by the heading.

File: guides/spin/pipeline-templates/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+spin pipeline-templates get --id myPipelineTemplate
+{
+  "id": "myPipelineTemplate"
+  "pipeline": {...}
+}
+```
+
+### Visualize a hydrated pipeline template with `plan`
+
+Visualize a hydrated pipeline template with `plan`:
```
Comment (dorbin) [inline guides/spin/pipeline-templates/index.md]: This one too.

**Review by dorbin — CHANGES_REQUESTED** [review]
Looks good, Jake.

Can you add to navigation.yml?

**Comment by jtk54** [comment]
Rebased and added an entry to navigation.yml.

**Review by dorbin — APPROVED** [review]
:+1:

**Pull request merged**